### PR TITLE
DOC-7211 export metrics|logs no creation date restriction

### PR DESCRIPTION
**Pull request by andf-crl**
Addresses: DOC-7211

- Removed mentions of creation date restriction from both `export-logs.md` and `export-metrics.md`.

Timing

- I will wait to merge until C C - 9 7 7 5 is merged.

Staging

[export-logs.md](https://deploy-preview-16516--cockroachdb-docs.netlify.app/docs/cockroachcloud/export-logs.html) | [export-metrics.md](https://deploy-preview-16516--cockroachdb-docs.netlify.app/docs/cockroachcloud/export-metrics.html)

### CODE REVIEW

**Review by thtruo — APPROVED** [review]
Nice @andf-crl this LGTM!

**Review by MichaelTrestman — APPROVED** [review]
lgtm

File: cockroachcloud/export-logs.md
Comment (abarganier) [inline cockroachcloud/export-logs.md:12]: Note: the limitation was removed for **GCP only**. AWS still has this limitation. We're working on fixing this right now, but until it's merged, deployed, and run, the AWS limitation still exists.

Can we rework this to only remove the GCP bit, leaving the AWS piece?

**Review by abarganier — CHANGES_REQUESTED** [review]

File: cockroachcloud/export-metrics.md
Comment (abarganier) [inline cockroachcloud/export-metrics.md:11]: Note: the AWS limitation also still exists for metric export.

We're hoping to resolve this soon, but I think we should scope this PR to only removing the text about the GCP limitation, and then follow up with another PR once the AWS limitation is removed.

**Comment by thtruo** [comment]
Update: this is now complete https://cockroachlabs.atlassian.net/browse/CC-9775 so we can fully remove the mentions of creation date restriction from both `export-logs.md` and `export-metrics.md`.

**Comment by thtruo** [comment]
One more request: in https://www.cockroachlabs.com/docs/cockroachcloud/export-logs.html we mention that Log Export is in Limited Preview. Now that we removed this creation date limitation for older clusters, we can treat Log Export as a Preview feature. Can we use the same language as in our Metric Export doc https://www.cockroachlabs.com/docs/cockroachcloud/export-metrics.html?

> This feature is in [preview](https://www.cockroachlabs.com/docs/v23.1/cockroachdb-feature-availability). This feature is subject to change. To share feedback and/or issues, contact [Support](https://support.cockroachlabs.com/hc/en-us).

**Comment by florence-crl** [comment]
I could not merge this PR. After consulting with @lnhsingh, I created and merged #17241.

**Pull request closed without merging**
